README.md

```diff
@@ -122,6 +122,8 @@ save("mygraph.jgz", g, "mygraph", compress=true)
 
 - **distance:** eccentricity, diameter, periphery, radius, center
```

Comment:
I'll do it when I get access to my laptop again, let me know of any other
improvement. Thank you.

-Júlio